<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MDSClass</class>
 <widget class="QMainWindow" name="MDSClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>400</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MDS</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="my_qlabel" name="label">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>30</y>
      <width>371</width>
      <height>311</height>
     </rect>
    </property>
    <property name="mouseTracking">
     <bool>true</bool>
    </property>
    <property name="frameShape">
     <enum>QFrame::Box</enum>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>600</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionForward"/>
   <addaction name="actionForward1"/>
   <addaction name="actionRestart"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionForward">
   <property name="icon">
    <iconset resource="mds.qrc">
     <normaloff>:/MDS/forward.png</normaloff>:/MDS/forward.png</iconset>
   </property>
   <property name="text">
    <string>forward</string>
   </property>
  </action>
  <action name="actionRestart">
   <property name="icon">
    <iconset resource="mds.qrc">
     <normaloff>:/MDS/restart.png</normaloff>:/MDS/restart.png</iconset>
   </property>
   <property name="text">
    <string>restart</string>
   </property>
  </action>
  <action name="actionForward1">
   <property name="icon">
    <iconset resource="mds.qrc">
     <normaloff>:/MDS/forward1.png</normaloff>:/MDS/forward1.png</iconset>
   </property>
   <property name="text">
    <string>forward1</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>my_qlabel</class>
   <extends>QLabel</extends>
   <header location="global">my_qlabel.h</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="mds.qrc"/>
 </resources>
 <connections/>
</ui>
